lua/nvim_lsp/nimls.lua

```diff
@@ -0,0 +1,43 @@
+local configs = require 'nvim_lsp/configs'
+local util = require 'nvim_lsp/util'
+
+configs.nimls = {
+  default_config = {
+    cmd = {"nimlsp",};
+    filetypes = {'nim'};
+    log_level = vim.lsp.protocol.MessageType.Warning;
+    settings = {};
+    root_dir = function(fname)
+      return util.find_git_ancestor(fname) or vim.loop.os_homedir()
+    end;
+  };
+  docs = {
```

Comment:
Would you add these lines?
```
  docs = {
    vscode = "kosz78.nim";
```

Reply:
Resolved in 9a5d9b385cc3a7967a6fde9a2b8eb4bd7215dd2a.